aws/internal/keyvaluetags/generators/updatetags/main.go

```diff
@@ -422,6 +432,16 @@ func ServiceTagInputTagsField(serviceName string) string {
 	}
 }
 
+// ServiceTagInputResourceTypeField determines the service tagging tags field requires a map[string]*string.
```

Comment:
Yet another permutation for the Kinesis Streams service.
Listing tags returns a `[]*kinesis.Tag` but the tags input to the `AddTags` API requires a `map[string]*string`.
Capture that here via another specialization.